examples/kms/vault/vault.yaml

```
@@ -100,6 +100,12 @@ items:
             bound_service_account_names="${SERVICE_ACCOUNTS}" \
             bound_service_account_namespaces="${SERVICE_ACCOUNTS_NAMESPACE}" \
             policies="${CLUSTER_IDENTIFIER}"
+
+        # disable iss validation
+        # from: external-secrets/kubernetes-external-secrets#721
+        vault write auth/kubernetes/config \
+          kubernetes_host="${K8S_HOST}" \
+          disable_iss_validation=true
```

May be stilll need to add `token_reviewer_jwt=@${SERVICE_ACCOUNT_TOKEN_PATH}/token \
          kubernetes_ca_cert=@${SERVICE_ACCOUNT_TOKEN_PATH}/ca.crt` here?

---

```suggestion
          token_reviewer_jwt=@${SERVICE_ACCOUNT_TOKEN_PATH}/token \
          disable_iss_validation=true
```
@nixpanic , need to add `token_reviewer_jwt` , works with it. 
Otherwise throws
 ```
 E0624 10:57:42.524988       1 utils.go:172] ID: 90 Req-ID: pvc-5bad374b-0c5c-414a-beb6-ba58b878e386 GRPC error: rpc error: code = InvalidArgument desc = invalid encryption kms configuration: failed creating new Vault Secrets: Error making API request.
URL: PUT http://vault.rook-ceph.svc.cluster.local:8200/v1/auth/kubernetes/login
Code: 400. Errors:
* missing client token
```

---

That is weird, because I did not need that in my environment. Also there is a `vault login` earlier in the script, so I wonder why I am not seeing this failure on my side.